kitsune/wiki/tests/test_templates.py

```diff
@@ -192,13 +192,29 @@ def test_document_fallback_with_translation_english_slug(self):
 
     def test_document_fallback_no_translation(self):
         """The document template falls back to English if no translation
-        exists."""
+        exists. The fallback message is only shown when there is any revision
+        ready for localization"""
         r = revision(save=True, content='Some text.', is_approved=True)
         url = reverse('wiki.document', args=[r.document.slug], locale='fr')
         response = self.client.get(url)
         doc = pq(response.content)
         eq_(r.document.title, doc('article h1.title').text())
 
+        # Fallback message is not shown because the revision is not ready for
+        # localization.
+        eq_(0, len(doc('#doc-pending-fallback')))
+        # Included content is English.
+        eq_(pq(r.document.html)('div').text(), doc('#doc-content div').text())
+
+        # Marking the revision ready for localization
+        r.is_ready_for_localization = True
+        r.save()
+
```

Comment:
As you inserted `document.is_localizable` here(1), This Warning will only show if the document is marked as localizable. so in test you should also mark the document is localizable.
it should be easy
   ```
    r.document.is_localizable = True
    r.document.save()
```
https://github.com/mozilla/kitsune/pull/2565/files#diff-31e8b5522cb01ad1ac16331cc8397eebR49